src/models/searchservice/response/FieldValueDirectAnswer.ts

```diff
@@ -7,11 +7,11 @@ import { Result } from './Result';
  *
  * @public
  */
-export interface FieldValueDirectAnswer extends DirectAnswer {
+export interface FieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
```

Comment:
Do we need the `T=unknown` here if we already have it in `DirectAnswer`?

Reply:
Yes, I tested it out and `T=unknown` is still needed else it errors out because we have to define the type variable in interface declaration to use T. But the `<T>` for DirectAnswer can be removed without errors.